tests/devapps/MauiMacAppWithBroker/MainPage.xaml.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace MauiMacAppWithBroker;
+
+using Microsoft.Maui.Controls;
+using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Broker;
+using System;
+using System.Runtime.InteropServices;
+
+public partial class MainPage : ContentPage
```

Comment:
can you please add this dev app to the pipeline, build it, and make sure the interop files are placed in the appropriate folders

Reply:
see here : https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/build/template-build-and-prep-automation.yaml#L63

Reply:
Can we track it with another work item? I created one https://identitydivision.visualstudio.com/Engineering/_workitems/edit/3168507
It's in my epic and will not be forgotten